src/renderers/dom/shared/__tests__/DOMPropertyOperations-test.js

```diff
@@ -203,6 +203,13 @@ describe('DOMPropertyOperations', () => {
       expect(stubNode.getAttribute('role')).toBe('<html>');
     });
 
+    it('should set values as boolean attributes', () => {
+      DOMPropertyOperations.setValueForProperty(stubNode, 'data-foo', true);
+      expect(stubNode.getAttribute('data-foo')).toBe('');
+      DOMPropertyOperations.setValueForProperty(stubNode, 'data-foo', false);
+      expect(stubNode.getAttribute('data-foo')).toBe(null);
+    });
+
```

Comment:
Is this true? No matter how I assign data attributes, they always appear, regardless of boolean values:

https://codepen.io/nhunzaker/pen/YVoYMV?editors=1010